source/adapters/level_zero/command_buffer.cpp

```diff
@@ -475,22 +528,18 @@ ur_exp_command_buffer_handle_t_::~ur_exp_command_buffer_handle_t_() {
 
   // Release additional signal and wait events used by command_buffer
   if (SignalEvent) {
-    CleanupCompletedEvent(SignalEvent, false);
     urEventReleaseInternal(SignalEvent);
```

Comment:
We are still ignoring the error codes returned by urEventReleaseInternal, zeFenceDestroy and urKernelRelease. Since we already have to move some of the code, I wonder if it would be better to just move everything from the destructor so that we can do proper error checking?